src/core/events.js

```diff
@@ -1,6 +1,7 @@
 import EventEmitter from 'eventemitter2'
 import store from '../store/store'
 import * as selectors from '../store/selectors'
+import { each } from 'lodash'
```

Comment:
No, it wouldn't be so straightforward. You would have to do something like:
```
 const objKeys = Object.keys(obj);
 objKeys.forEach(/*your code*/);
```
I won't deny is more verbose that lodash each, but still not tremendously so and this is native code that doesn't depend on any library and that in the long run will cause fewer problems. But like I said, not a review per se, just a topic worth discussing further.